app/models/proxy.rb

```diff
@@ -10,6 +10,8 @@ class Proxy < ApplicationRecord # rubocop:disable Metrics/ClassLength
   include GatewaySettings::ProxyExtension
   include ProxyConfigAffectingChanges::ModelExtension
 
+  audited :allow_mass_assignment => true
```

Comment:
How about `audited allow_mass_assignment: true` ?

Reply:
Good point, it was removed from audited before awhile so we can also remove it everywhere.
https://github.com/collectiveidea/audited/issues/223